.gitignore

```diff
@@ -8,7 +8,7 @@ projects-tmp/
 *.parquet
 examples/pipeline/results/
 ploomber-venv
-
+doc/*/**/*.md
```

Comment:
@yafimvo why do we need to ignore it?

Reply:
Added it to ignore some artifacts in my local env.
Rolled it back.